### PR TITLE
Enable transactional update tests for zkvm

### PR DESCRIPTION
**Pull request by JRivrain**
Test has to be adjusted for s390

- Related ticket: https://progress.opensuse.org/issues/65301
- Verification run: https://openqa.suse.de/tests/4306347

### CODE REVIEW

File: lib/transactional.pm [hunk 1]
```diff
@@ -39,10 +40,13 @@ sub process_reboot {
         microos_reboot $trigger;
     } else {
         power_action('reboot', observe => !$trigger, keepconsole => 1);
-
-        # Replace by wait_boot if possible
-        assert_screen 'grub2', 100;
-        send_key 'ret';
+        if (check_var('ARCH', 's390x')) {
+            reconnect_mgmt_console(timeout => 500);
```
Comment (b10n1k) [inline lib/transactional.pm:44]: Have you tried `boot/reconnect_mgmt_console.pm`? I wonder if the module works why do we need to mess with this file. if the `boot/reconnect_mgmt_console.pm` module doesnt work then we need to adjust it. WDYT?

Comment (JRivrain) [reply lib/transactional.pm:44]: No, as you can see in the test run, we call this function multiple times, for each reboot.

**Comment by JRivrain** [comment]
> IMO if `boot/reconnect_mgmt_console.pm` works it will make the schedule more transparent and we will keep clean this module, letting it do one thing and not mix it with not related code.

No, as you can see in the test run, we call need to call reconnect_mgmt_console multiple times, for each reboot.

**Comment by rwx788** [comment]
We should find some time to improve rebooting for all the scenarios on s390x, as we have those hacks in each case where we need to handle the reboot.

**Pull request merged**